### PR TITLE
ADM-362 T8 - [frontend] [Metrics] Real done

### PR DESCRIPTION
**Pull request by spadio9587**
Context/Background

In order to see 

Acceptance Criteria

Real Done title

DONE and CANCELLED as default selection

With All option in the box (not show in the inputBox)

When Real Done section turns to empty, it will appear error message - Must select which you want to consider as Done

All Context fields from your jira board are candidates

Option field

### CODE REVIEW

File: frontend/src/components/Metrics/MetricsStep/RealDone/style.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export const Divider = styled('div')({
+  padding: '0.4rem',
+  borderLeft: `0.4rem solid ${theme.main.backgroundColor}`,
+  margin: '1rem 0',
+})
+
+export const Title = styled('span')({
+  fontFamily: theme.typography.fontFamily,
+})
```
Comment (tingwang1234) [inline frontend/src/components/Metrics/MetricsStep/RealDone/style.tsx:12]: maybe Divider and Title could be extracted to a common file

File: frontend/cypress/pages/metrics.ts [hunk 2]
```diff
@@ -62,11 +62,17 @@ class Metrics {
   goMetricsStep() {
     cy.contains('Next').click()
     cy.contains('Crews Setting').should('exist')
+    cy.contains('Real Done').should('exist')
```
Comment (guzhongren) [inline frontend/cypress/pages/metrics.ts:65]: Should not contain any assertion in POM, 
Ref to recent blog 'Selenium之POM 设计模式'

and PLZ remove the above assertion

File: frontend/cypress/pages/metrics.ts [hunk 3]
```diff
@@ -62,11 +62,17 @@ class Metrics {
...
-    cy.contains('Next').click()
-    cy.contains('Crews Setting').should('exist')
+  checkRealDoneOption() {
+    cy.contains('Consider as Done').siblings().click()
+    cy.get("[type='checkbox']").uncheck()
+
+    cy.contains('DONE').click()
+    cy.contains('CANCELLED').click()
+
+    cy.get('div.MuiBackdrop-root.MuiBackdrop-invisible.MuiModal-backdrop').click({ force: true })
```
Comment (guzhongren) [inline frontend/cypress/pages/metrics.ts:75]: Recommend using other info to do assertion,
because css is not stable when the dependency upgrading

**Comment by guzhongren** [comment]
Your PR Summary does not follow our default convention, WHY?

**Review by guzhongren — CHANGES_REQUESTED** [review]
Your PR Summary does not follow our default convention, WHY?

**Review by guzhongren — COMMENTED** [review]
Update by comments

File: frontend/cypress/pages/metrics.ts (hunk 2 above)
Comment (spadio9587) [reply frontend/cypress/pages/metrics.ts:65]: Got that, will fix that soon!

File: frontend/cypress/pages/metrics.ts (hunk 3 above)
Comment (spadio9587) [reply frontend/cypress/pages/metrics.ts:75]: Will fix that soon about assertion

File: frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import {
```
Comment (Teresasasa) [inline frontend/src/components/Metrics/MetricsStep/RealDone/index.tsx:1]: this file is almost same as the index file of `Crews` components. Could we abstract a common component in order to decrease the  duplicate code?

File: frontend/src/components/Metrics/MetricsStep/index.tsx [hunk 5]
```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import { Crews } from '@src/components/Metrics/MetricsStep/Crews'
 import { useAppSelector } from '@src/hooks'
 import { selectUsers } from '@src/context/board/jiraVerifyResponse/jiraVerifyResponseSlice'
+import { RealDone } from '@src/components/Metrics/MetricsStep/RealDone'
 
 export const MetricsStep = () => {
   const users = useAppSelector(selectUsers)
-
+  const realDoneOptions = ['DONE', 'CANCELLED']
```
Comment (Teresasasa) [inline frontend/src/components/Metrics/MetricsStep/index.tsx:9]: I am not very sure about why real done options using `Done` and `Cancelled` as default value. I know this is written in the card. But what if the board don't have a cancelled status? What if the board have more other status in the doneColumn? I mean the status in the done column may vary from boards to boards. I think it probably need to use the data queried from configuration.

**Review by guzhongren — APPROVED** [review]

**Review by Eason-T — APPROVED** [review]

**Review by tingwang1234 — APPROVED** [review]
LGTM

**Pull request merged**